docs/dev/guide/cross-chain-tutorial.md

```diff
@@ -322,7 +313,7 @@ async function main() {
   // governance contract
   const wallet = new ethers.Wallet("<WALLET-PRIVATE-KEY>", l1Provider);
 
-  const governance = new ethers.Contract(GOVERNANCE_ADDRESS, GOVERNANCE_ABI, wallet);
+  const govcontract = new Contract(GOVERNANCE_ADDRESS, GOVERNANCE_ABI, wallet);
```

Comment:
why need to change the name? If you like `govcontract` more, then it should be `govContract`

Reply:
Because the code snippets  in the docs  tutorial should be the same as code in the repo cross-chain repo: https://github.com/matter-labs/cross-chain-tutorial/blob/main/counter/scripts/increment-counter.ts

Reply:
I think we should change the repo as well then

Reply:
But since we need to deploy asap, ok